pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -60,6 +60,10 @@
 , zsh
 , # codeium-nvim dependencies
   codeium
+, # codesnap-nvim dependencies
+  clang
+, libclang
```

Comment:
```suggestion
```

It might be useless to explicitly add `libclang` as that's handled automatically by `rustPlatform.bindgenHook`

Reply:
Indeed it seems to work without it. I removed it.